submissions/ArthurGorbenko/HTML5-Form/index.html

```diff
@@ -0,0 +1,487 @@
+                <option value="german">German</option>
+                <option value="сhinese">Chinese</option>
+              </select>
+            </span>
+            <div class="wrapper_form-selects">
+              <span class="select_from select_to"></span>
+              <span class="form-select_custom form-selects_bottom">
+                <select name="" id="" class="button form-selects">
+                  <option value="arabian">Arabian</option>
+                  <option value="hindi">Hindi</option>
+                  <option value="french">French</option>
+                </select>
+              </span>
+              <span class="form-selects_bottom form-select_custom">
+                <select name="" id="" class="button form-selects">
+                  <option value="hindi">Hindi</option>
+                  <option value="arabian">Arabian</option>
+                  <option value="french">French</option>
+                </select>
+                <div class="form-select_remove"></div>
+              </span>
+              <button type="button" class="button form-select_add">
+                <span class="form-select_add_text">Add</span>
+              </button>
+            </div>
+          </section>
+          <div class="form-wrapper_radio_tone_urgent">
+            <section class="form-select_tone">
+              <span class="form-sections_headers_text form-select_tone_header"
+                >Select your tone</span
+              >
+              <input
+                type="radio"
+                name="tone"
+                id="casual"
+                class="visually__hidden"
+              />
+              <label for="casual" class="button form-select_tone_radio_label">
+                <span class="form-select_tone_icon casual"
+                  >Friendly/casual</span
+                >
+              </label>
+              <input
+                type="radio"
+                name="tone"
+                id="corporate"
+                class="visually__hidden"
+              />
+              <label
+                for="corporate"
+                class="button form-select_tone_radio_label"
+              >
+                <span class="form-select_tone_icon pro">Corporate/pro</span>
+              </label>
+              <input
+                type="radio"
+                name="tone"
+                id="formal"
+                class="visually__hidden"
+              />
+              <label for="formal" class="button form-select_tone_radio_label">
+                <span class="form-select_tone_icon formal">Media/formal</span>
+              </label>
+            </section>
+            <section class="form-range_urgent">
+              <span
+                class="form-sections_headers_text form-sections_headers_text_urgent"
+              >
+                Tell us how urgent this job is
+              </span>
+              <div class="wrapper_form-range">
+                <input
```

Comment:
This input is required.
Also, check colors)

Reply:
Done.